manifests/s/Shodo-1/2/0/1.2.0/Shodo-1.2.0.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Shodo-1.2.0

```

```suggestion
PackageIdentifier: PJe.Sjodo
```
Please update file name and path to match this change.